packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.28+3
+
+* Update the e2e tests to use `onCameraIdle` instead of using an arbitrary delay.
```

Comment:
It seems very strange to me that we have to bump a version for a test-only update.

Reply:
We do update the versions for any changes including tests. I haven't really questioned this. I can't think of a good reason to do so. @csells or @amirh might know.

Reply:
I'm curious if it's just that it's easier to force this for all changes rather than trying to detect changes that are test/infra only.